monitoring/README.md

```diff
@@ -422,6 +422,10 @@ git pull
 # deploy the manual changes seen, as well as the new intended changes
 ./grafana/bin/upload-dashboards.sh
 
+# deploy library panel updates
```

Comment:
is there a way we can enforce this? / does anything bad happen if someone versions without any updates?

Reply:
No, they'll just cause unneeded version bump noise that will be included in the next PR/commit that saves grafana dashboards.